.github/workflows/ci.yml

```diff
@@ -288,9 +288,9 @@ jobs:
         run: |
           cd docs/build/html
```

Comment:
I’d really recommend using https://github.com/marketplace/actions/deploy-to-github-pages

There’s much less to maintain, code is more succinct, and someone else is improving the functionality and making it cross platform.

Reply:
@akaszynski thanks for your input. We added a task to take this comment into account